proms-resources/frontend/src/main/frontend/src/TOU.json

```
@@ -0,0 +1,80 @@
+    "This Agreement and applicable UHN policies and procedures constitute the sole and entire agreement between you and UHN regarding the Application and supersedes all prior and contemporaneous understandings, agreements, representations and warranties, both written and oral, regarding such subject matter.\\\n",
+    "###EPRO PRIVACY POLICY\\\n",
+    "This privacy policy governs your use of the software application EPRO (**Application**) that was created by Smart Cancer Care and the UHN (UHN). EPRO assists patients in reporting their symptoms and other health-related data to their healthcare providers. Based on the data you enter and the status of your health, you will get feedback and self-care instructions. It is not intended to replace your current care, but to supplement it with additional support.\\\n",
+    "####What information does the Application collect and how is it used**\\\n",
+    "####User Provided Information\\\n",
+    "The Application obtains the information you provide when you download and register with the Application.",
+    "When you register with us and use the Application, you generally provide (a) email address, password and other registration information; (b) information about your condition, which may include self-reported symptom information and (c) information you provide us when you contact us for technical help with the application. UHN may also use your contact information (phone or email) to share important information about the Application and required updates.\\\n",
+    "####Automatically Collected Information\\\n",
+    "In addition, the Application may collect certain information automatically, including, but not limited to, the type of mobile device you use, your mobile devices unique device ID, the IP address of your mobile device, your mobile operating system, the type of mobile Internet browsers you use, and information about the way you use the Application.\\\n",
+    "####Do third parties see and/or have access to information obtained by the Application**\\\n",
+    "Only aggregated, anonymized data is periodically transmitted to external services to help us improve the Application and our service. We will share your information with third parties only in the ways that are described in this privacy statement. We may disclose User Provided and Automatically Collected Information: * As required by law, such as to comply with a subpoena, or similar legal process; * With our trusted services providers who work on our behalf, do not have an independent use of the information we disclose to them, and have agreed to adhere to the rules set forth in this privacy statement and the Personal Health Information Protection Act, 2004 and any subsequent amendments.\\\n",
+    "####Use of your Information by your Health Care Team\\\n",
+    "The information you enter into this Application will be provided to your healthcare team and will become part of your medical record.\\\n",
+    "####What are my rights**\\\n",
```

There was supposed to be a question mark at the end of this line, that probably got confused with a special character and got replaced with `**` in the formatting process. There are several other occurrences of this replacement across the text.
```suggestion
    "####What are my rights?",
```

---

Fixed in 8c43d6e